Source/WebCore/inspector/agents/InspectorNetworkAgent.h

```diff
@@ -70,6 +70,10 @@ class InspectorNetworkAgent : public InspectorAgentBase, public Inspector::Netwo
     WTF_MAKE_TZONE_ALLOCATED(InspectorNetworkAgent);
     WTF_MAKE_NONCOPYABLE(InspectorNetworkAgent);
 public:
+    struct ConfigurationOptions {
```

Comment:
can we move this closer to where it's used in `protected:`?

also this is much nicer!  great idea :)

Reply:
or even better why not move this to something like `NetworkResourcesData::Settings` so that `InspectorNetworkAgent` can just pass it along?